config/sidekiq.yml

```diff
@@ -13,3 +13,8 @@
   - ["notifications_low", 2]
   - ["notifications", 1] # TODO: this queue can be deleted in a future PR once high/medium/low are in place
   - ["default", 1]
+
+:schedule:
```

Comment:
ah, I thought this was a paid feature of sidekiq that we're not paying for. Perhaps I've misunderstood, I thought what we could do was use our existing cron job which called `feeds:all` and convert the individual feeds to cronjobs which were invoked by the existing cronjob.

Reply:
This uses a separate gem https://github.com/moove-it/sidekiq-scheduler :)

Reply:
Thinking this through, I'm not sure if it is safe to schedule directly in the Sidekiq pods? in most environments there are 2 sidekiq pods. So presumably the scheduling will run on both pods and hence the job will be scheduled twice? It might be safer to get a rake task triggered by a cronjob to add the tasks to the sidekiq queue, and then whichever worker is free can pick it up. And we would have no worries if we want to increase the number of sidekiq pods in the future.

Reply:
it would be good to test on dev : e.g. schedule a job once and see if it runs twice?